### DISCUSSION TITLE
Covestro Deutschland AG

### DISCUSSION THREAD

**Discussion by qcserestipy** [aquasecurity/trivy#6089] [Adopters]
### How do you use Trivy?
We use the trivy operator to scan for vulnerable images in our production K8s cluster. We use vulnerability reports and config audit reports as source for policy decisions of our Open Policy Agent Gatekeeper to prevent further potentially vulnerable workloads running in our cluster. Additionally, we use the trivy scanner in our harbor container registry to ensure safe deployments to various production workloads such as apache airflow or gitlab runner.

### Which targets are you scanning with Trivy?

- [x]  Container Image
- [x]  Filesystem
- [ ] Git Repository
- [ ] Virtual Machine Image
- [x] Kubernetes
- [ ] AWS
- [ ] SBOM

### What kind of issues are scanning with Trivy?
 
- [ ] Software Bill of Materials (SBOM)
- [x]  Known vulnerabilities (CVEs)
- [x] IaC issues and misconfigurations
- [x] Sensitive information and secrets
- [ ] Software licenses

**Comment by AnaisUrlichs** [comment]
Thank you so much for sharing! 😊
